lib/ModelMixins/CatalogFunctionJobMixin.ts

```diff
@@ -289,6 +291,7 @@ function CatalogFunctionJobMixin<
     }
     protected async forceLoadMapItems() {}
 
+    @action
```

Comment:
Does this need an `@action` here?
Sorry I'm not a fan of wrapping async functions in actions - I find it a bit unintuitive where the action starts and then the promise starts with async/await syntax

Reply:
scratch this too